src/code/z_kaleido_scope_call.c

```diff
@@ -70,21 +70,22 @@ void KaleidoScopeCall_Update(PlayState* play) {
                 pauseCtx->savePromptState = PAUSE_SAVE_PROMPT_STATE_APPEARING;
                 pauseCtx->state = (pauseCtx->state & 0xFFFF) + 1; // PAUSE_STATE_WAIT_BG_PRERENDER
             }
-        } else if (pauseCtx->state == PAUSE_STATE_8) {
+        } else if (pauseCtx->state == PAUSE_STATE_GAME_OVER_REQUEST) {
 #if OOT_DEBUG
             R_HREG_MODE = HREG_MODE_UCODE_DISAS;
             R_UCODE_DISAS_LOG_MODE = 3;
 #endif
 
             R_PAUSE_BG_PRERENDER_STATE = PAUSE_BG_PRERENDER_SETUP;
             pauseCtx->mainState = PAUSE_MAIN_STATE_IDLE;
-            pauseCtx->savePromptState = PAUSE_SAVE_PROMPT_STATE_APPEARING;
-            pauseCtx->state = (pauseCtx->state & 0xFFFF) + 1; // PAUSE_STATE_9
-        } else if ((pauseCtx->state == PAUSE_STATE_WAIT_BG_PRERENDER) || (pauseCtx->state == PAUSE_STATE_9)) {
+            pauseCtx->savePromptState = PAUSE_SAVE_PROMPT_STATE_APPEARING; // redundant for game over
```

Comment:
what is meant by this comment? why is it redundant?

Reply:
all in all, this comment is redundant, but here is the explanation:

thing is, there are two if blocks in `z_kaleido_scope_call.c`, for normal pause and the game over variant, and they are completely identical, as if copypasted or written with macros: https://github.com/zeldaret/oot/pull/2283#discussion_r1827652667

but `PAUSE_SAVE_PROMPT_STATE_APPEARING` bears meaning only for `PAUSE_STATE_SAVE_PROMPT`:
```
// z_kaleido_scope.c
case PAUSE_STATE_SAVE_PROMPT:
    switch (pauseCtx->savePromptState) {
        case PAUSE_SAVE_PROMPT_STATE_APPEARING:
```

whereas `PAUSE_STATE_GAME_OVER_*` has no such sub-state machine